### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 10

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#507] [Esercizi M-Z]
![es.10](https://github.com/user-attachments/assets/4851f014-870c-4a1b-b10b-dc9ca27284d0)

**Comment by Vinz002** [comment]
![1](https://github.com/user-attachments/assets/260bf966-05b8-44ec-aaef-d89ae56653c3)
![2](https://github.com/user-attachments/assets/94f21dcb-8435-453f-82da-ad229cf7e9a0)
![3](https://github.com/user-attachments/assets/e6e9f892-bc65-4427-a609-07c3c5d97288)
![4](https://github.com/user-attachments/assets/a730e8cf-eff8-4e70-a070-51da6a14231b)
